tests/Unit/DI/config_00.neon

```diff
@@ -0,0 +1 @@
+monolog:
```

Comment:
![Screenshot 2019-05-30 at 11 55 06](https://user-images.githubusercontent.com/538058/58625002-ccf69a80-82d1-11e9-953b-399201fd9394.png)

Please move it to `tests/fixtures`